build.sh

```diff
@@ -7,7 +7,8 @@
 runtime=${1:-linux-x64}
 shift
 
-options=(--configuration='Release' --framework='net6.0' --self-contained='false' --output='./bin' /p:SuppressNETCoreSdkPreviewMessage=true /p:PublishTrimmed=false --runtime=$runtime)
+options=(--configuration='Release' --framework='net6.0' --self-contained='false' --output='./bin' \
+/p:SuppressNETCoreSdkPreviewMessage=true /p:PublishTrimmed=false --runtime=$runtime -p:SourceRevisionId=$(git rev-parse --short HEAD))
```

Comment:
```suggestion
    /p:SuppressNETCoreSdkPreviewMessage=true /p:PublishTrimmed=false --runtime=$runtime -p:SourceRevisionId=$(git rev-parse --short HEAD))
```
It's more obvious its continuing last line this way.